### PR TITLE
use higher ports for attachment http srv

### PR DESCRIPTION
**Pull request by mmaxim**
Move port range higher up to decrease change of conflicting with other programs.

cc @songgao in case you care.

### CODE REVIEW

**Comment by mmaxim** [comment]
cc @malgorithms too

**Comment by songgao** [comment]
Thanks for the heads up. I'll vendor this into KBFS once it's merged.

Curious question: We had 1001 ports to choose from. Are there programs that use all of them?

**Comment by mmaxim** [comment]
It wasn't a problem for us, but if we bound to the port before another program that uses a port in that range, then that program would get blocked. Someone reported that they couldn't start Cassandra since it uses 7000.

**Comment by mmaxim** [comment]
Actually I don't want to use ephemeral range, I'm worried the app will conflict with itself on mobile when we need to be able to get the same port back.

**Comment by songgao** [comment]
I see. Makes sense.

File: go/chat/attachment_httpsrv.go [hunk 1]
```diff
@@ -66,7 +66,7 @@ func NewAttachmentHTTPSrv(g *globals.Context, fetcher types.AttachmentFetcher, r
 	r := &AttachmentHTTPSrv{
 		Contextified: globals.NewContextified(g),
 		DebugLabeler: utils.NewDebugLabeler(g.GetLog(), "AttachmentHTTPSrv", false),
-		httpSrv:      libkb.NewHTTPSrv(g.ExternalG(), libkb.NewPortRangeListenerSource(7000, 8000)),
+		httpSrv:      libkb.NewHTTPSrv(g.ExternalG(), libkb.NewPortRangeListenerSource(16723, 18000)),
```
Comment (songgao) [inline go/chat/attachment_httpsrv.go]: What's special with 16723?

Comment (mmaxim) [reply go/chat/attachment_httpsrv.go]: Seems random enough to not be used by anyone.

**Pull request merged**